code/game/area/Space Station 13 areas.dm

```diff
@@ -1173,6 +1173,38 @@ proc/process_adminbus_teleport_locs()
 	name = "\improper Holodeck - Space"
 	jammed=SUPER_JAMMED
 
+/area/holodeck/source_firingrange
+	name = "\improper Holodeck - Firing Range"
+	jammed=SUPER_JAMMED
```

Comment:
Put spaces on all of these.  Yes I know your eyes are rolling.  No I don't like saying it.